test/parsing/errors/newWithoutArgs-detailed.good

```diff
@@ -0,0 +1,3 @@
+─── error in newWithoutArgs.chpl:5 [NewWithoutArgs] ───
+  Type in 'new' expression is missing its argument list.
```

Comment:
Strictly speaking, isn't the new expression itself missing the argument list, not the type? A type itself may not have arguments, e.g.
```
record R{ var x: int; }
```
the type `R` is just that; the new expression is what needs arguments.

Reply:
I agree; this is copied from the previous error message. I'll change it to simply "'New' expression is missing its argument list.".